src/qililab/circuit_transpiler/circuit_transpiler.py

```diff
@@ -107,7 +196,7 @@ def translate(  # pylint: disable=too-many-locals, too-many-branches
                 # apply gate schedule
                 for gate_event in gate_schedule:
                     # find bus
-                    bus = self.platform._get_bus_by_alias(gate_event.bus)  # pylint: disable=protected-access
+                    bus = self.platform.get_bus_by_alias(gate_event.bus)  # pylint: disable=protected-access
```

Comment:
If it is not longer a protected method, do we still need the pylint ignore command here?